### ISSUE TITLE
Randomizer() i.e. next level random search

### ISSUE THREAD

**Issue by mikkokotila** [autonomio/talos#84]
It looks like possibly the lowest hanging fruit in terms of providing the user with options for the optimization strategy, and improving the results that come from uniform standards, have to do with quasirandom (low discrepancy) sequences. 

The issue with uniform random function i.e. pseudorandom function is "clumping" where luck plays an important role. The goal of the quasirandom approaches (e.g. Latin Hypercube, Sobol, Halton...) is to minimize clumping. 

One of the more interesting variations here seems to be *Latin hypercube with a sudoku-like constraint*. [[link]](https://github.com/Technologicat/sudoku_lhs/blob/master/sudoku_lhs/sudoku.py)

Basically what we seem to need here is a way to perform a quasirandom shuffle (instead of the pseudorandom one we use now). That library is not available it seems, so it might have to be built. My goal would be to make available for the user several quasirandom shuffle options.

**Comment by ruojol** [comment]
Is this allready available in dev branch?

**Comment by mikkokotila** [comment]
You can get it in daily-dev for now. Most likely will merge with dev this week. The PR is already there, but the tests (basically pep8) suggest that I better clean the code before merging.

**Comment by mikkokotila** [comment]
Note that Latin improved takes a very long time to, so I'm going to remove that.

**Comment by C0NTRIBUTE** [comment]
It would be nice to be able to input a random function (that takes in a 1-d array of integers obviously) through the random_method argument in scan. That way if the user is already working with random numbers with something like `mlk_random`, they can just use whatever method they like.

**Comment by mikkokotila** [comment]
This was done some time ago, so closing.
